exasol/toolbox/templates/github/workflows/checks.yml

```diff
@@ -95,9 +95,37 @@ jobs:
       - name: Run type-check
         run: poetry run nox -s type-check
 
-  tests-job:
+  Security:
+    name: Security Checks (Python-${{ matrix.python-version }})
```

Comment:
Is there a section explaining the security checks? 
Which mechanism is used to identify security issues?

Reply:
Not yet, but this is also not part of the change. Still, I agree with you. Therefore I asked @Jannis-Mittenzwei to follow up with some documentation for the security.

That said, I don't think the "how" needs to matter so much for the user, rather the results.

Reply:
OK for me.